wamr-compiler/CMakeLists.txt

```diff
@@ -49,7 +49,12 @@ add_definitions(-DWASM_ENABLE_MODULE_INST_CONTEXT=1)
 
 if (WAMR_BUILD_LLVM_LEGACY_PM EQUAL 1)
   add_definitions(-DWASM_ENABLE_LLVM_LEGACY_PM=1)
-endif()
+endif ()
+
+if (WAMR_BUILD_LINUX_PERF EQUAL 1)
+  message ("-- Enable linux perf support")
+  add_definitions(-DWASM_ENABLE_LINUX_PERF=1)
+endif ()
```

Comment:
Had better add the macro directly (remove if/endif), and just using `wamrc --enable-linux-perf` to enable the feature for AOT file. Or we need to compile wamrc individually.